### DISCUSSION TITLE
Add agro ecological zones dataset

### DISCUSSION THREAD

**Discussion by KilimoTT** [lsc-hubs/kenya-hub#16] [General]
When we are talking about soil, there are other parameters like agro-ecological zones. I would appreciate if AEZ datasets are also captured in the hub.

**Comment by pvgenuchten** [comment]
The agro-ecological zones are available, we could highlight them in a thematic map

https://kenya.lsc-hubs.org/stac/collections/metadata:main/items/gaez
